packages/graphql-api/src/utils/authenticated-resolver.ts

```diff
@@ -1,8 +1,8 @@
-export const authenticated = (func: any) => async (
-  root: any,
-  args: any,
-  context: any,
-  info: any
+export const authenticated = <R, A, C, I, T>(func: (root: R, args: A, context: C, info: I) => T | Promise<T>) => async (
```

Comment:
when using generics in the codebase we try to be more verbose so it's easier to understand, for example, `root` will be `CustomRoot` instead of `R` and `args` will be `CustomArgs` instead of `A`.